app.js

```diff
@@ -24,6 +25,9 @@ app.set('views', __dirname + '/views');
 app.set('view engine', 'ejs');
 app.set('port', process.env.PORT || 8080);
 
+// Set the region
+AWS.config.update({region: 'us-west-2'});  //TODO change to envvar
```

Comment:
Did you want to change this before merging?

Reply:
Since S3 is global it doesn't matter much for this specific use case. I can probably remove the comment.